apps/veil/src/pages/tournament/ui/round/current-voting-results/index.tsx

```diff
@@ -0,0 +1,165 @@
+          },
+        ],
+        symbol: 'um',
+        penumbraAssetId: { inner: new Uint8Array([1]) },
+        coingeckoId: 'um',
+        images: [],
+        name: 'um',
+        description: 'um',
+      }),
+    },
+    case: 'knownAssetId',
+  },
+});
+
+export const CurrentVotingResults = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [page, setPage] = useState(1);
+  const [limit, setLimit] = useState(10);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setIsLoading(false);
+    }, 1000);
+  }, []);
+
+  const totalPages = 100;
+  const totalVotes = 10000;
+
+  const data = [
+    {
+      symbol: 'USDC',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 2777,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'OSMO',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 1337,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'ATOM',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 1000,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'USDC',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 7,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'OSMO',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 3,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'ATOM',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 10,
+      estimatedIncentive: valueView,
+    },
+  ];
+
+  return (
+    <Card>
+      <div className='flex flex-col p-3 gap-4'>
+        <Text xxl color='text.primary'>
+          Current Voting Results
+        </Text>
+        <Density compact>
+          <div className='grid grid-cols-5 h-auto overflow-auto'>
+            <div className='grid grid-cols-subgrid col-span-5'>
+              <TableCell heading>Asset</TableCell>
+              <TableCell heading>Gauge Value</TableCell>
+              <TableCell heading>Casted Votes</TableCell>
+              <TableCell heading>Estimated Incentive</TableCell>
+              <TableCell heading>Vote</TableCell>
+            </div>
+
+            {isLoading ? (
+              Array.from({ length: 6 }).map((_, i) => (
+                <div key={i} className='grid grid-cols-subgrid col-span-5'>
+                  {Array.from({ length: 5 }).map((_, j) => (
+                    <TableCell loading key={j}>
+                      <div className='w-24 h-5'>
+                        <Skeleton />
+                      </div>
+                    </TableCell>
+                  ))}
+                </div>
```

Comment:
suggestion (non-blocking): would be easier to add `loading={isLoading}` to every TableCell instead of creating new table. Especially having TableRow, it needs only one prop passing. Also a good thing in this approach is that columns will stay of fixed width if not changed

Reply:
How do you make sure to render a couple of rows when there's no data yet?

Reply:
Similar to this: https://github.com/penumbra-zone/web/blob/main/apps/veil/src/pages/tournament/ui/previous-epochs.tsx#L43